library/spdm_requester_lib/libspdm_req_get_key_pair_info.c

```diff
@@ -20,6 +20,10 @@ typedef struct {
     uint16_t public_key_info_len;
     uint8_t assoc_cert_slot_mask;
     uint8_t public_key_info[SPDM_MAX_PUBLIC_KEY_INFO_LEN];
+    uint32_t pqc_asym_algo_cap_len;
+    uint32_t pqc_asym_algo_capabilities;
```

Comment:
For 1.4 `PqcAsymAlgoCapabilities` only uses two bytes.

Reply:
Yes, 2 bytes is enough. But I want to align to base_pqc_asym.

Also, we still support the peer to use 2 bytes.